### DISCUSSION TITLE
Padding does not appear on pdf

### DISCUSSION THREAD

**Discussion by FarahKhandwala** [davidgohel/flextable#607] [Q&A] [answered]
Hello, I am new to flextable and having trouble rendering my padding settings to pdf.  The following appears correctly in the viewer:

  df <- structure(list(
    type = c("glop glop", "glop glop" , "glop glop", "pas glop pas glop", "pas glop pas glop"), 
    what = c("Group", "Age", "Residence", "Smoker", "Europe"), 
    `1` = c(63, 25, 25, 15, 15), 
    `2` = c(23, 53, 53, 74, 11),
    `3` = c(85, 22, 43, 13, 15)
  ), 
  row.names = c(NA, -5L), 
  class = c("data.frame"))
  
  as_grouped_data(df, groups = "type") %>% 
    as_flextable() %>% 
    add_footer_lines("Observed event") %>%
    set_header_labels(what = "") %>% 
    color(part = "footer", color = "#800000") %>%
    bold( bold = TRUE, part="header") %>% 
    padding(i = ~ is.na(type), padding.left = 20) %>% 
    bold(i = ~ !is.na(type)) %>%
    autofit()

![image](https://github.com/davidgohel/flextable/assets/87337820/d42610e6-1f49-4713-b4b6-a0f09fd907c9)
 
but when I render to pdf through quarto, I lose the indent. (In fact if I try any padding settings they do not seem to do anything):
![image](https://github.com/davidgohel/flextable/assets/87337820/9c074671-c896-4925-9141-58f703eabbb8)

**Comment by davidgohel** [comment, marked as answer]
PDF does not support padding, you can use `\t` instead 

```
library(flextable)
library(magrittr)

df <- structure(list(
  type = c("glop glop", "glop glop" , "glop glop", "pas glop pas glop", "pas glop pas glop"), 
  what = c("Group", "Age", "Residence", "Smoker", "Europe"), 
  `1` = c(63, 25, 25, 15, 15), 
  `2` = c(23, 53, 53, 74, 11),
  `3` = c(85, 22, 43, 13, 15)
), 
row.names = c(NA, -5L), 
class = c("data.frame"))

as_grouped_data(df, groups = "type") %>%
  as_flextable() %>%
  add_footer_lines("Observed event") %>%
  set_header_labels(what = "") %>%
  color(part = "footer", color = "#800000") %>%
  bold( bold = TRUE, part="header") %>%
  prepend_chunks(i = ~ is.na(type), 
                 as_chunk("\t")) %>%
  bold(i = ~ !is.na(type)) %>%
  autofit()

```
